src/app/services/contentful/tf-contentful.module.ts

```diff
@@ -0,0 +1,10 @@
+import { ContentfulService } from './tf-contentful.service';
+import { NgModule } from '@angular/core';
+
+@NgModule({
+  imports: [],
+  exports: [],
+  declarations: [ContentfulService],
```

Comment:
service is not required to be declared but provided. So please add this to providers and remove it from `app_providers`